src/data-mgt/kafka/streams/kcca-streams/src/main/java/net/airqo/serializers/JsonSerializer.java

```diff
@@ -0,0 +1,37 @@
+package net.airqo.serializers;
```

Comment:
Could we relocate this java code to the `java` subdirectory of the `data-mgt` micro? @NoahNsimbe